src/app/asset-page/artstor-viewer/artstor-viewer.component.html

```diff
@@ -48,21 +59,14 @@
         <button class="btn btn--icon btn--prev" id="previousButton-{{ osdViewerId}}"><</button><span class="btn-divider"></span><button class="btn btn--icon btn--next" id="nextButton-{{ osdViewerId}}">></button>
         </div> -->
         <!-- Edge pagination arrow style -->
-        <div [class.aiw-hidden]="!isMultiView" class="asset-viewer__edge-arrows" id="imagePageButtons-{{ osdViewerId }}">
-        <button class="btn btn--icon btn--prev" id="previousButton-{{ osdViewerId}}"><i class="icon icon-prev--white"></i></button><button class="btn btn--icon btn--next" id="nextButton-{{ osdViewerId}}"><i class="icon icon-next--white"></i></button>
+        <div [class.aiw-hidden]="!isMultiView" class="asset-viewer__edge-arrows" id="imagePageButtons-{{ osdViewerId }}" tabindex="-1">
```

Comment:
I think there might be something going on with the previous button. I can't seem to get it to appear locally using this example: https://localhost:3000/#/asset/1002971919

Reply:
Removing *ngIf="multiViewPage == multiViewCount" fixed the most immediate problem but that conditional may've been serving some other purpose that I didn't notice.

Reply:
The *ngIf is a leftover of previous implementation. I should remove it, doing that now.